dune_client/client_async.py

```diff
@@ -134,6 +141,14 @@ async def get_result(self, job_id: str) -> ResultsResponse:
             return ResultsResponse.from_dict(response_json)
         except KeyError as err:
             raise DuneError(response_json, "ResultsResponse", err) from err
+        
+    async def get_latest_result(self, query: Query) -> ResultsResponse:
+        """GET results from Dune API for `query_id`"""
```

Comment:
`GET the latest results for a query_id without having to execute the query again.`
you can also include the link to the API docs if you'd like